testing/synergy/missing_sentinel.patch

```diff
@@ -0,0 +1,78 @@
+--- src/lib/platform/XWindowsScreen.cpp.orig	2017-02-27 21:26:49.297547435 +0800
```

Comment:
Change this to a `-p1` style patch, by changing it to `--- a/src/lib/platform/XWindowsScreen.cpp` and `+++ b/src/lib/platform/XWindowsScreen.cpp` in the patch header.